src/app/routes/regex/index.test.js

```diff
@@ -176,3 +177,24 @@ describe('mediaRadioAndTvRegexPathsArray', () => {
     shouldNotMatchInvalidRoutes(invalidRoutes, mediaRadioAndTvRegexPathsArray);
   });
 });
+
+describe('mediaAssetPageRegexPath', () => {
+  const validRoutes = [
+    '/pidgin/tori-49450859',
+    '/pidgin/tori-49450859.amp',
+    '/yoruba/media-49450859',
+    '/yoruba/media-49450859.amp',
+    '/punjabi/international-49567825',
+    '/punjabi/international-49567825.amp',
```

Comment:
Can we add an example like `kyrgyz/sapar-tv-48695523` with the additional `-` and then add an example using `_` which we can make up like so: `mundo/test_underscore-12345678`

Reply:
will add it, Thanks Phil